pyfar/classes/audio.py

```diff
@@ -569,6 +584,12 @@ def __init__(
             used for energy signals, such as impulse responses.
         comment : str
             A comment related to `data`. The default is ``None``.
+        complex : bool
+            A flag which indicates if: in the case of raw data in the time
+            domain, the data are real or complex-valued, and in the case of
+            raw data in the frequency domain, the data are the Fourier
+            spectrum of a real or complex-valued time signal. The default
+            is ``False``.
```

Comment:
In my opinion, this is a little difficult to understand.. Maybe it is easier to describe when refering to the domain parameter or at least split it up into two sentences?